doc/tools/doc_status.py

```diff
@@ -250,15 +250,16 @@ def generate_for_class(c):
         for tag in list(c):
             if tag.tag in ['methods']:
                 for sub_tag in list(tag):
-                    methods.append(sub_tag.find('name'))
+                    methods.append(sub_tag.attrib['name'])
             if tag.tag in ['members']:
                 for sub_tag in list(tag):
                     try:
-                        methods.remove(sub_tag.find('setter'))
-                        methods.remove(sub_tag.find('getter'))
+                        if(sub_tag.attrib['setter'].startswith('_') == False):
```

Comment:
sadly necessary because the methods starting with a "_" are not mentioned in the docs as methods, and if the setter is not in the docs, the getter will not be filtered out, one could remove the second if case as a thrown exception there does nothing too terrible i guess, just left it there for readability